src/infinite-hits/infinite-hits.ts

```diff
@@ -82,7 +82,13 @@ export class NgAisInfiniteHits extends BaseWidget {
     public instantSearchParent: any
   ) {
     super('InfiniteHits');
-    this.createWidget(connectInfiniteHitsWithInsights, { escapeHits: true });
+  }
+
+  ngOnInit() {
+    this.createWidget(connectInfiniteHitsWithInsights, {
+      transformItems: this.transformItems,
+    });
```

Comment:
Do we also want to support `escapeHTML` and `showPrevious`? Those are the 3 options that the connector accepts.

Reply:
IIRC we agreed about the alignment of the options but not new features that take time (like `showPrevious`). The `escapeHTML` should be added though it's a one-liner.

Reply:
@yannickcr showPrevious was not implemented as part of the PR that introduced show previous on infinite hits: https://github.com/algolia/angular-instantsearch/pull/486/files#diff-04dba89bfc1cca2b9a95e2e98f92a6a1L70

Reply:
escapeHTML exposed here 88fe28f
Not sure it's relevant to add a story as we have no dataset that currently allows to showcase it.

Reply:
@tkrugg I'm not sure to understand everything yet. What is the benefit to add `showPrevious` here?

Reply:
it's just one of the optional parameters the connector accepts. 
https://github.com/algolia/instantsearch.js/blob/develop/src/connectors/infinite-hits/connectInfiniteHits.ts#L118

Reply:
Oh, I see. I think this is an oversight. I will make a fix.

Reply:
I'll merge this as is so you can add it :)